probeinterface/probe.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+
+_possible_electrode_shapes = ['circle', 'square', 'rect']
+
+
+class Probe:
+    """
+    Class to handle the geometry of one probe.
+    
+    Handle mainly electrode position.
+    
+    Can be 2D or 3D.
+    
+    Can handle also optionally the shape of eletrode and the shape of the probe.
+    
+    
+    """
+    def __init__(self, ndim=2, si_units='um'):
+        """
+        
+        Parameters
+        ----------
+        ndim: 2 or 3
+            handle 2D or 3D probe
+        
+        si_units: 'um', 'mm', 'm'
+        
+        """
+        self.ndim = ndim
+        self.si_units = si_units
+        
+        # electrode position and shape : handle with arrays
+        self.electrode_positions = None
```

Comment:
We should add `self.electrode_axes`

This should be a numpy array with dimensions (num_electrodes, 2, ndim). This defines the electrod plane (2d or 3d) where the electrode lies. 

this will be necessary to allow 3d rotations.

Reply:
yep.